lib/kubeclient/http_proxy_wrapper.rb

```diff
@@ -0,0 +1,49 @@
+module Kubeclient
+  module Common
+    # Wrapper around Net:HTTP to support k8s services
+    class HttpProxyWrapper
+      def initialize(uri, options)
+        @uri = uri
+        @options = options
+      end
+
+      def get(path)
```

Comment:
I thought about creating them when they'll be needed.
The general idea is to get an HTTP object from kubeclient which have already the proper path prefix (e.g api/v1/proxy/namespaces/default/services/image-scanner:5001-tcp), the problem is that the Net:HTTP object doesn't hold a path, so I created this one, which actually wraps the HTTP:Net object and a path.